pkgs/development/python-modules/pydateinfer/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, unittestCheckHook
+, pytz
+, pyyaml
+, argparse
+}:
+
+buildPythonPackage rec {
+  pname = "pydateinfer";
+  version = "0.3.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "wdm0006";
+    repo = "dateinfer";
+    rev = "0.3.0,"; # yes the comma is required, this is correct name of git tag
+    hash = "sha256-0gy7wfT/uMTmpdIF2OPGVeUh+4yqJSI2Ebif0Lf/DLM=";
+  };
+
+  propagatedBuildInputs = [
+    pytz
+  ];
+
+  preCheck = "cd dateinfer";
+  nativeCheckInputs = [
+    unittestCheckHook
+    pyyaml
+    argparse
+  ];
+  pythonImportsCheck = [ "dateinfer" ];
+
+  meta = with lib; {
+    description = "Infers date format from examples";
+    homepage = "https://pypi.org/project/pydateinfer/";
+    license = licenses.asl20;
```

Comment:
Is there a changelog in the source repo by any chance ?

Reply:
There is no changelog file and no realeses

Reply:
We can theoretically add it as https://github.com/wdm0006/dateinfer/compare/v0.2.0..0.3.0%2C but I'm not sure that it is the right way to define changelog